src/Hydra/Protocol/Overview.tex

```diff
@@ -80,28 +81,42 @@ \subsection{The Coordinated Head protocol}
 transactions that have been processed into it as the snapshot's multi-signature
 is now evidence that this state once existed during the head evolution.
 
-\red{Besides processing ``normal'' transactions, participants can also request
-  to withdraw some UTxO they can spend from the Head and make it available on main
-  chain, or deposit some UTxO they can spend from L1 and make it available within the Head.
-  These processes are called decrement, via the \mtxDecrement{} transaction,
-  and increment, via the \mtxIncrement{} transaction.}
+\subsubsection{Updating an open head}
+
+Besides processing ``normal'' transactions, participants can also request to
+withdraw some UTxO they can spend from the Head and make it available on main
+chain via a \mtxDecrement{}~\ref{sec:decrement-tx} transaction --- the overall process is also called
+``decommit''.
+
+\red{The inverse of locking more UTxO on the mainchain and making it available
+	within the head is called (incremental) ``commit''. First, anyone may create a
+	deposit of one or more UTxO using a \mtxDeposit{}~\ref{sec:deposit-tx}
+	transaction. The head participants will observe this deposit and, once
+	settled, request off-chain agreement to include the deposited UTxO in the form
+	of a snapshot. With that agreement, an \mtxIncrement{}~\ref{sec:increment-tx}
+	transaction can be created and used to update the head state on the mainchain.
+	A deadline is associated with the deposit, which ensures that the UTxO is
+	locked up long enough to be safely consumed into the head without risk of
+	double spending. Should a deposit have not been picked up in time, a
+	\mtxRecover{}~\ref{sec:recover-tx} transaction allows anyone to unlock the
+	original UTxO.}
```

Comment:
How is it ensured that somebody doesn't recover something I tried to deposit and steel my funds?

Reply:
So locked UTxO would go back to their original address and datum hashes would make sure they go to the correct address that locked them? And somebody would just be able to pay for the recover tx but not own the produced UTxO.

Reply:
Correct. The deposit validator ensures that the ("original") UTxO is unlocked exactly as they were locked.